allennlp/nn/beam_search.py

```diff
@@ -471,19 +475,25 @@ def __init__(
         beam_size: int = 10,
         per_node_beam_size: int = None,
         sampler: Sampler = None,
+        min_steps: int = 0,
```

Comment:
I think making this an `Optional[int]` is more clear.

```suggestion
        min_steps: Optional[int] = None,
```

Reply:
Ok, I pushed this change. I had to change the code to make sure the value is valid, too.